packages/app/src/systems/Settings/pages/ChangePassword/ChangePassword.tsx

```diff
@@ -78,17 +99,30 @@ export function ChangePassword() {
               control={control}
               name="newPassword"
               label="New Password"
+              hideError
               render={({ field }) => (
-                <InputPassword
-                  {...field}
-                  onBlur={() => {
-                    trigger();
-                    field.onBlur();
-                  }}
-                  css={styles.input}
-                  aria-label="New Password"
-                  placeholder="Type your new password"
-                />
+                <PasswordStrength
+                  onOpenChange={() => setPasswordTooltipOpened(true)}
+                  password={field.value || ''}
+                  open={passwordTooltipOpened}
+                  minLength={8}
+                  onChangeStrength={(strength: string) =>
+                    setValue('strength', strength)
+                  }
+                >
+                  <InputPassword
+                    {...field}
+                    onBlur={() => {
+                      trigger();
+                      field.onBlur();
+                      setPasswordTooltipOpened(false);
+                    }}
+                    onFocus={() => setPasswordTooltipOpened(true)}
+                    css={styles.input}
+                    aria-label="New Password"
+                    placeholder="Type your new password"
+                  />
+                </PasswordStrength>
```

Comment:
We should abstract the password input with a strength tooltip to a component to avoid having these controls open and close and more than one place. Also will make the code cleaner. With the new component, we can simplify here and `[CreatePassword.tsx](https://github.com/FuelLabs/fuels-wallet/pull/520/files#diff-542a5cd353b0bf260d112740ba6b26d4d7b457cf7faf0278f5ea2a7a5cf85787)`